static/app/views/replays/detail/layout.tsx

```diff
@@ -0,0 +1,263 @@
+import BaseResizePanel from 'react-resize-panel';
+import styled from '@emotion/styled';
+
+import ErrorBoundary from 'sentry/components/errorBoundary';
+import ReplayTimeline from 'sentry/components/replays/breadcrumbs/replayTimeline';
+import ReplayView from 'sentry/components/replays/replayView';
+import space from 'sentry/styles/space';
+import useFullscreen from 'sentry/utils/replays/hooks/useFullscreen';
+import theme from 'sentry/utils/theme';
+
+import Breadcrumbs from './breadcrumbs';
+import FocusArea from './focusArea';
+import FocusTabs from './focusTabs';
+
+type Layout =
+  /**
+   * ### Sidebar
+   * ┌───────────────────┐
+   * │ Timeline          │
+   * ├──────────┬────────┤
+   * │ Details  > Video  │
+   * │          >        │
+   * │          >^^^^^^^^┤
+   * │          > Crumbs │
+   * │          >        │
+   * └──────────┴────────┘
+   */
+  | 'sidebar'
+  /**
+   * ### Topbar
+   *┌────────────────────┐
+   *│ Timeline           │
+   *├───────────┬────────┤
+   *│ Video     │ Crumbs │
+   *│           │        │
+   *├^^^^^^^^^^^^^^^^^^^^┤
+   *│ Details            │
+   *│                    │
+   *└────────────────────┘
+   */
+  | 'topbar';
```

Comment:
I was trying to use layouts like 'horizontal' and 'vertical' but i think these names are clearer. The diagrams helped me to sort out where the grabber should live, so i left them in.